protocols/example/crud/shoppingcart/shoppingcart.proto

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019 Lightbend Inc.
```

Comment:
@ralphlaude Is this a 1-to-1 copy of the "original" shoppingcart.proto?

Reply:
@viktorklang it is not a 1-to-1 copy. here there is the `RemoveShoppingCart` request with the corresponding endpoint which can be used to completely delete a shopping cart.